src/modules/nodejs.rs

```diff
@@ -12,7 +12,13 @@ use crate::context::Context;
 ///     - Current directory contains a `package.json` file
 ///     - Current directory contains a `node_modules` directory
 pub fn segment(context: &Context) -> Option<Segment> {
-    let is_js_project = context.dir_files.iter().any(has_js_files);
+    let js_criteria = find_file::Criteria {
```

Comment:
It looks to me like we'll eventually need all three `find_file::Criteria` to be vectors.
We should probably plan ahead for that. 🤔

Reply:
👍